apl/scalar-functions/string-functions/trim.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: trim
+description: 'This page explains how to use the trim function in APL.'
+---
+
+The `trim` function removes all leading and trailing matches of a specified regular expression from a string. You use it when you want to clean up input values by stripping unwanted characters, whitespace, or tokens at the start or end of strings. This is useful in log analysis, cleaning up OpenTelemetry trace attributes, or standardizing values in security logs where input formats vary.
```

Comment:
Nah, no regular expressions involved, trim/trimstart/trimend/ all just use a "cutset" and remove those characters from both/start/end, internally they use golangs [Trim/TrimLeft/TrimRight](https://pkg.go.dev/strings#Trim)